doc/source/api-reference/qibo.rst

```diff
@@ -560,6 +560,34 @@ Thermal relaxation channel
 
 _______________________
 
+Noise Model
+-----------
+
+In Qibo is possible to create a custom noise model using the
+class :class:`qibo.noise.NoiseModel`. This enables the user to create
+circuits where the noise is gate and qubit dependent.
+
+For more information on the use of :class:`qibo.noise.NoiseModel` see
+:ref:`How to perform noisy simulation? <noisemodel-example>`
+
+.. autoclass:: qibo.noise.NoiseModel
+    :members:
+    :member-order: bysource
+
+The quantum errors available to build a noise model are the following:
```

Comment:
We could have a separate subsection for the quantum errors here:

```suggestion
Quantum errors
^^^^^^^^^^^^^^^

The quantum errors available to build a noise model are the following:
```

Reply:
I agree. It is more readable this way.